### PR TITLE
Add the IATI Datastore as Tool

### PR DESCRIPTION
**Pull request by siemvaessen**
Add the IATI Datastore as Tool

### CODE REVIEW

**Comment by andylolz** [comment]
Looks good to me.

Repo is here:
https://github.com/zimmerman-zimmerman/iati.cloud

Contributor guidelines are here:
https://github.com/zimmerman-zimmerman/iati.cloud#can-i-contribute

File: _projects/13_iati_datastore.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: IATI Datastore
+tags:
+    - tools
+link: https://iatidatastore.iatistandard.org/ 
+
```
Comment (andylolz) [inline _projects/13_iati_datastore.md]: Needs a `---` just here.

**Comment by andylolz** [comment]
Repo is here:
https://github.com/zimmerman-zimmerman/iati.cloud

Contributor guidelines are here:
https://github.com/zimmerman-zimmerman/iati.cloud#can-i-contribute

File: _projects/13_iati_datastore.md (hunk 1 above)
Comment (siemvaessen) [reply _projects/13_iati_datastore.md]: where?

Comment (andylolz) [reply _projects/13_iati_datastore.md]: Line 6. The frontmatter needs to be sandwiched in `---`s, [as shown in the documentation](https://jekyllrb.com/docs/front-matter/).

**Comment by andylolz** [comment]
@siemvaessen fix is wrong, I’m afraid.

if you tick the "Allow edits from maintainers" box on the pull request, I can fix this.

**Pull request merged**